catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/FormulaEditorTest.java

```diff
@@ -71,10 +73,23 @@ public void numericValuesTest() {
 
 		//typeText not working for formula editor, so use CustomActions.typeInValue
 		onFormulaEditor()
-				.performEnterNumber(12345.678);
+				.performEnterNumber(12345.678)
+				.performCloseAndSave();
+	}
+
+	@Category({Cat.AppUi.class, Level.Smoke.class})
+	@Test
+	public void wrappedInBracketsTest() {
```

Comment:
write an instrumented unit test for this. (there are already a handful testing the getResultForComputeDialog method)
and remove the UI test.